includes/class-amp-theme-support.php

```diff
@@ -118,7 +126,7 @@ public static function init() {
 			$args = array_shift( $support );
 			if ( ! is_array( $args ) ) {
 				trigger_error( esc_html__( 'Expected AMP theme support arg to be array.', 'amp' ) ); // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_trigger_error
```

Comment:
This appears on the AMP Settings page `/wp-admin/admin.php?page=amp-options`
>( ! ) Warning: array_merge(): Argument #2 is not an array in /srv/www/loc/public_html/wp-content/plugins/amp/includes/class-amp-theme-support.php on line 189
>( ! ) Notice: Expected AMP theme support arg to be array. in /srv/www/loc/public_html/wp-content/plugins/amp/includes/class-amp-theme-support.php on line 128

This occurs with Twenty Seventeen. That would make sense, as it doesn't do `add theme_support( 'amp' )`, so `$support` is `array()`

And `$args` is null, because `$args = array_shift( $support );`